jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/services/asn1/Asn1CommandManager.java

```diff
@@ -94,14 +102,19 @@ public Asn1CommandManager(OdeProperties odeProperties) {
 
    }
 
-   public void depositToDDS(String asdBytes) throws Asn1CommandManagerException {
-      try {
-         depositor.deposit(asdBytes);
-         EventLogger.logger.info("Message Deposited to SDW: {}", asdBytes);
-         logger.info("Message deposited to SDW: {}", asdBytes);
-      } catch (DdsRequestManagerException e) {
-         String msg = "Failed to deposit message to SDW";
-         throw new Asn1CommandManagerException(msg, e);
+   public void depositToSdw(String asdBytes) throws Asn1CommandManagerException {
+      
+      if (this.odeProperties.shouldDepositSdwMessagesOverWebsocket()) {
+         try {
+            depositor.deposit(asdBytes);
+            logger.info("Deposited message to SDW directly via websocket {}", asdBytes);
+         } catch (DdsRequestManagerException e) {
+            String msg = "Failed to deposit message to SDW";
+            throw new Asn1CommandManagerException(msg, e);
+         }
+      } else {
+         stringMessageProducer.send(this.getDepositTopic(), null, asdBytes);
+         logger.info("Published message to SDW deposit Kafka topic {}", asdBytes);
       }
```

Comment:
Missing EventLoger: `EventLogger.logger.info("Message Deposited to SDW: {}", asdBytes);`